### PR TITLE
ci(release): update pipeline to use an Action

### PR DESCRIPTION
**Pull request by bnb**
Updates the release pipeline to use @jasonetco's GitHub Action. I think I've done it right, but eyes would be appreciated. I'm curious if I've missed anything in deleting the release script (looked through it and don't think I have). Also curious if I've implemented the action correctly.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "url": "https://github.com/pkgjs/meet.git"
   },
   "scripts": {
+    "build": "npx @vercel/ncc build && npx convert-action",
```
Comment (wesleytodd) [inline package.json:14]: Would be great to install `convert-action` as a dev, and then convert the ziet version dev dep to vercel.  I know that is not required, but there are good reasons to be explicit about your deps.

**Comment by wesleytodd** [comment]
I think this PR would replace this, right?

https://github.com/pkgjs/meet/pull/79

cc @Eomm

**Comment by bnb** [comment]
I don't think we need to close that PR since it's unrelated to the files being changed here.

**Comment by bnb** [comment]
Going to merge.

**Pull request merged**